regularExpressionCheatsheet.Rmd

```diff
@@ -0,0 +1,19 @@
+# Cheatsheet
```

Comment:
Hi, this title will appear as the chapter name of your project on the website. Please choose a more appropriate one.

> Choose a short, descriptive, human readable title for your project as your title will show up in the table of contents